docs/upctl_server_create.md

```diff
@@ -0,0 +1,57 @@
+## upctl server create
+
+Create a server
+
+```
+upctl server create [flags]
+```
+
+### Options
+
+```
+      --avoid-host int                  Use this to make sure VMs do not reside on specific host. Refers to value from host -attribute. Useful when building HA-environments.
+      --host int                        Use this to start a VM on a specific host. Refers to value from host -attribute. Only available for private cloud hosts.
+      --boot-order string               The boot device order, disk / cdrom / network or comma separated combination.
+      --user-data string                Defines URL for a server setup script, or the script body itself.
+      --cores int                       Number of cores. Use only when defining a flexible ("custom") plan.
+      --memory int                      Memory amount in MiB. Use only when defining a flexible ("custom") plan.
+      --title string                    Visible name.
+      --hostname string                 Hostname.
+      --plan string                     Server plan name. See "server plans" command for valid plans. Set --cores and --memory for a flexible plan. (default "1xCPU-2GB")
+      --os string                       Server OS to use (will be the first storage device). Set to empty to fully customise the storages. (default "Ubuntu Server 20.04 LTS (Focal Fossa)")
+      --os-storage-size os              OS storage size in GiB. This is only applicable if os is also set. Zero value makes the disk equal to the minimum size of the template.
+      --zone string                     Zone where to create the server.
+      --password-delivery string        Defines how password is delivered. Available: email, sms (default "none")
+      --simple-backup string            Simple backup rule. Format (HHMM,{dailies,weeklies,monthlies}). Example: 2300,dailies
+      --time-zone string                Time zone to set the RTC to. (default "UTC")
```

Comment:
RTC could be made more clear, I think? (Maybe something like RTC/VM system time?)